source/clear-linux/reference/collaboration/structure-formatting.rst

```diff
@@ -0,0 +1,483 @@
+.. _structure-formatting:
+
+Structure and formatting
+########################
+
+Content should be organized to support scanning. Consistent organization,
+formatting, and writing style helps readers quickly find what they need and to
+understand the content more effectively. This document describes our
+organization and formatting guidelines.
+
+Refer to :ref:`writing-guide` to learn how we keep our documents clear and
+concise.
+
+.. contents:: :local:
+   :depth: 1
+
+Markup
+******
+
+Our documentation is written in the reStructuredText markup language, using
+Sphinx roles and directives. We use Sphinx to generate the final documentation.
+You can read more about reStructuredText and Sphinx on their respective
+websites:
+
+* `Sphinx documentation`_
+* `reStructuredText Primer`_
+
+You can view the content directly in the .rst markup files, or you can generate
+the HTML content by installing and running the documentation locally. To run the

```

Comment:
"running the documentation locally" sounds funny to me. I'd prefer building.

Reply:
Done.